lib/services/ios-debug-service.ts

```diff
@@ -72,14 +74,15 @@ class IOSDebugService implements IDebugService {
         }).future<void>()();
     }
 
-    private emulatorDebugBrk(): IFuture<void> {
+    private emulatorDebugBrk(shouldBreak? : boolean): IFuture<void> {
```

Comment:
Of course,
https://github.com/telerik/mobile-cli-lib/blob/master/declarations.d.ts#L133
https://github.com/telerik/mobile-cli-lib/blob/master/declarations.d.ts#L413
https://github.com/telerik/mobile-cli-lib/blob/master/declarations.d.ts#L431
https://github.com/NativeScript/nativescript-cli/blob/master/lib/declarations.ts#L170
https://github.com/NativeScript/nativescript-cli/blob/master/lib/declarations.ts#L320

But anyway, it depends on you to decide which approach to take. As I've said, this is not a merge stopper :)